CHARTER.md

```diff
@@ -1,72 +1,72 @@
 
-#### 1. Mission and Scope of the Technical Initiative
+#### 1. Mission and Scope of the Working Group
 
-- a. The mission of the Technical Initiative is to provide a forum for our members and the broader community to collaborate on Global Cybersecurity-related legislation, frameworks, and standards which facilitate conformance to regulatory requirements by open source projects and their consumers.
+- a. The mission of the Working Group is to provide a forum for our members and the broader community to collaborate on Global Cybersecurity-related legislation, frameworks, and standards which facilitate conformance to regulatory requirements by open source projects and their consumers.
 
-- b. The scope of the Technical Initiative includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration and the creation of other artifacts that support the mission.
+- b. The scope of the Working Group includes collaborative development under the Technical Initiative License (as defined herein) supporting the mission, including organizing collaboration activities, defining best practices, documentation, testing, integration and the creation of other artifacts that support the mission.
 
 #### 2. Technical Steering Committee
 
-- a. The Technical Steering Committee (the "TSC") will be responsible for all oversight of the Technical Initiative. 
+- a. The Working Group will be considered synonymous with the Technical Steering Committee (the "TSC" elsewhere in this document) and will be responsible for all oversight of the Working Group. 
 
-- b. The TSC voting members are initially the Technical Initiative’s Maintainers. The Maintainers will be documented in the Technical Initiative repository. The TSC is responsible for determining the future process for defining voting members of the TSC, and any such alternative approach will also be documented appropriately.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
+- b. Members of the working group will be documented in the Working Group's repository. Anyone may join the Working Group by making a pull request on the [MEMBERS.md](https://github.com/ossf/wg-globalcyberpolicy/blob/main/governance/MEMBERS.md) document. A member of the group's leadership team will approve these requests. Any meetings of the Working Group are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
```

Comment:
"A member of the group's leadership will approve these requests."  Always?  Or are there times they might not be approved (previous behaviour, OLAF issues, etc.?).  If so, "A member of the group's leadership will review these requests and approve unless further information is required."